src/screens/OrganizationDashboard/OrganizationDashboard.tsx

```diff
@@ -120,42 +122,90 @@ function organizationDashboard(): JSX.Element {
               </Row>
             ) : (
               <Row style={{ display: 'flex' }}>
-                <Col xs={6} sm={4} className="mb-4">
+                <Col
+                  xs={6}
+                  sm={4}
+                  role="button"
+                  className="mb-4"
+                  onClick={(): void => {
+                    history.push(`/orgpeople/id=${organizationId}`);
```

Comment:
Can you create variable for the url/link e.g. 
```ts
const peopleLink: string = `/orgpeople/id=${organizationId}`;
```

Reply:
I added the variable for this.